src/createServer.js

```diff
@@ -1,8 +1,66 @@
 'use strict';
 
+const http = require('http');
+const fs = require('fs');
+const path = require('path');
+
 function createServer() {
-  /* Write your code here */
-  // Return instance of http.Server class
+  return http.createServer((req, res) => {
+    const normalizedURL = new URL(req.url, `http://${req.headers.host}`);
+    const fileName = normalizedURL.pathname.slice(1);
+
+    res.setHeader('Content-Type', 'text/plain');
+
+    if (!req.url.startsWith('/file')) {
+      res.statusCode = 400;
+      res.end('Invalid request');
+
+      return;
+    }
+
+    if (!req.url.startsWith('/file/')) {
+      res.end('Hint message: Use /file/ to access files');
+
+      return;
+    }
+
+    if (req.url.includes('//')) {
+      res.statusCode = 404;
+      res.end('Invalid path');
+
+      return;
+    }
+
+    const requestedPath = fileName.slice(5);
```

Comment:
The slicing operation `fileName.slice(5)` assumes that the URL always starts with '/file/'. Ensure that this assumption holds true in all cases, or consider using a more flexible approach to extract the requested path.